<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="1" failures="0" skipped="0" tests="1" time="0.091" timestamp="2022-09-03T16:37:27.583791" hostname="LAPTOP-N7P03V5Q"><testcase classname="" name="Tests.test_assistant" time="0.000"><error message="collection failure">ImportError while importing test module 'C:\Users\tajex\PycharmProjects\Dashboard\Tests\test_assistant.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
..\..\AppData\Local\Programs\Python\Python310\lib\importlib\__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
Tests\test_assistant.py:3: in &lt;module&gt;
    from assistant import *
assistant.py:13: in &lt;module&gt;
    from bs4 import BeautifulSoup as bs4
E   ImportError: cannot import name 'BeautifulSoup' from 'bs4' (unknown location)</error></testcase></testsuite></testsuites>